packages/forms-web-app/src/dynamic-forms/validator/string-validator.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable no-unused-vars */
+const { body } = require('express-validator');
+const BaseValidator = require('./base-validator.js');
+
+class StringValidator extends BaseValidator {
+	regex = {
+		regex: '',
+		regexMessage: ''
+	};
+	minLength = {
+		minLength: 0,
+		minLengthMessage: ''
+	};
+	maxLength = {
+		maxLength: 0,
+		maxLengthMessage: ''
+	};
+	constructor({ regex, minLength, maxLength } = {}) {
+		super();
+
+		if (!regex && !minLength && !maxLength) throw new Error('Nothing to do!');
```

Comment:
Not sure we want to throw an error as it's a valid scenario that we might not have any validation on a string

Reply:
Then don't use the validator in the first place?

Reply:
We might want to only evaluate max length and the allowed characters and not worry about a min length for example, or maybe we only want to do a regex evaluation.

Reply:
Please read the unit tests.